### DISCUSSION TITLE
Seeking review of essay: 'The Second Renaissance: A Very Short Introduction'

### DISCUSSION THREAD

**Discussion by catherinet1** [life-itself/community#1068] [Research]
Hi everyone,

here is the complete first draft of our introductory essay about the Second Renaissance, which we shared about in the Life Itself Research Community call last week: https://docs.google.com/document/d/14I_9XtMJZqvvFJy1lyQsR3F63O8-CAmNO-d_rx2AyMU/edit#heading=h.fqys5t1jxymv.

Please feel free to share your questions and thoughts directly in the Google Doc as comments.

Curious to hear feedback, particularly on a) what strongly resonates with you? and b) where do you feel friction or tension with what is expressed here?

**Comment by rufuspollock** [comment]
Thanks @catherinet1 and looking forward to reviewing.

**Comment by asimong** [comment]
Thanks @catherinet1 for this welcoming invitation to review and comment. I have done so! I'm also imagining it might be good to have some scheduled open conversation time where points raised can be talked over? My own comments are at several levels, not all related to the text itself.
